content/id/docs/tasks/tools/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Instalasi Peralatan"
```

Comment:
```suggestion
title: "Menginstal Peralatan"
```

Reply:
Skipped this. It should be a verb.

Reply:
fixed in [4886c64](https://github.com/kubernetes/website/commit/4886c6443e7d06bf85252c47527bf9dc5471e737)